store/streaming/file/service_test.go

```diff
@@ -17,15 +18,14 @@ import (
 	"github.com/cosmos/cosmos-sdk/codec"
 	codecTypes "github.com/cosmos/cosmos-sdk/codec/types"
 	"github.com/cosmos/cosmos-sdk/store/types"
-	sdk "github.com/cosmos/cosmos-sdk/types"
```

Comment:
We shall need the import "sdk" down below for sdk.BigEndianToUint64

Reply:
thanks